pkg/kv/kvnemesis/validator.go

```diff
@@ -532,6 +533,56 @@ func (v *validator) processOp(op Operation) {
 		if v.buffering == bufferingSingle {
 			v.checkAtomic(`deleteRangeUsingTombstone`, t.Result)
 		}
+	case *AddSSTableOperation:
+		if resultIsErrorStr(t.Result, "key range .* outside of bounds of range") {
```

Comment:
That's a `RangeKeyMismatchError`, can we please use `errors.HasType` and friends whenever possible? We shouldn't have lost the structure here.

Reply:
Done. Just followed the example of the other operations here, and assumed there was a reason for it, but this is better.